README.md

```diff
@@ -943,3 +943,45 @@ Join block geometries to employment attributes:
 ```
 $ tippecanoe-json-tool -c in_wac_S000_JT00_2015.csv tl_2010_18157_tabblock10.sort.json > blocks-wac.json
 ```
+
+Error returns
```

Comment:
Should this be useful for checking programmatic progress/success/failure? If so do the error codes need to be contiguous or can we split them into ranges by category of failure e.g. 100-199, 200-299 etc?